webapi/_test_data/ontologies/newton-onto.ttl

```diff
@@ -0,0 +1,122 @@
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix bibo: <http://purl.org/ontology/bibo/> .
+@prefix event: <http://purl.org/NET/c4dm/event.owl#> .
+@prefix knora-base: <http://www.knora.org/ontology/knora-base#> .
+@prefix salsah-gui: <http://www.knora.org/ontology/salsah-gui#> .
+@prefix beol: <http://www.knora.org/ontology/0801/beol#> .
+@prefix biblio: <http://www.knora.org/ontology/0801/biblio#> .
+@base <http://www.knora.org/ontology/0801/newton#> .
+
+@prefix : <http://www.knora.org/ontology/0801/newton#> .
+
+<http://www.knora.org/ontology/0801/newton> rdf:type owl:Ontology ;
+    rdfs:label "The Newton-Project ontology" ;
+    knora-base:attachedToProject <http://rdfh.ch/projects/yTerZGyxjZVqFMNNKXCDPF> .
+
+##########################################################
+#
+# PROPERTIES
+#
+##########################################################
+
+
+### ###########################################
+### newton:hasAbbreviatedTitle
+
+:hasAbbreviatedTitle rdf:type owl:ObjectProperty ;
+
+
+                rdfs:label "Abkürzung des Titels"@de ,
+                           "Abbreviation of Title"@en ;
+
+                rdfs:comment """Represents the abbreviated title of the letter."""@en ;
+
+                knora-base:objectClassConstraint knora-base:TextValue ;
+                knora-base:subjectClassConstraint :letter ;
+
+                salsah-gui:guiElement salsah-gui:SimpleText ;
+
+                rdfs:subPropertyOf knora-base:hasValue .
+
+### ###########################################
+### newton:newtonProjectID
+
+:newtonProjectID rdf:type owl:ObjectProperty ;
+
+
+                rdfs:label "Newton Project ID"@de ,
+                           "Newton Project ID"@en ;
+
+                rdfs:comment """Represents the ID of the letter in The Newton Project."""@en ;
+
+                knora-base:objectClassConstraint knora-base:TextValue ;
+                knora-base:subjectClassConstraint :letter ;
+
+                salsah-gui:guiElement salsah-gui:SimpleText ;
+
+                rdfs:subPropertyOf knora-base:hasValue .
+
+### ###########################################
+### newton:hasFacsimiles
+
+:hasFacsimiles rdf:type owl:ObjectProperty ;
+
+                   knora-base:objectClassConstraint knora-base:UriValue ;
+
+                   knora-base:subjectClassConstraint :letter ;
+
+                   rdfs:subPropertyOf knora-base:hasValue ;
+
+                   rdfs:label "Faksimile"@de ,
+                              "Facsimile"@en ;
+
+                   rdfs:comment """Die Faksimilen von deisem Brief ist vorhanden in"""@de ,
+                              """The Facsimiles of this letter are available in"""@en .
+
+
+##########################################################
+#
+# RESOURCES
+#
+##########################################################
+
+### ###########################################
+### newton:letter
+
+:letter rdf:type owl:Class ;
+
+        rdfs:subClassOf beol:letter ,
```

Comment:
Would it make more sense to have a shared base class of `newton:letter` and `beol:letter`, rather than making `newton:letter` a subclass of `beol:letter`? See [Shared Ontologies](https://docs.knora.org/paradox/02-knora-ontologies/introduction.html#shared-ontologies).

Reply:
done :)